.github/pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+<!-- markdownlint-disable-next-line MD041 -->
+### Checklist
+
+- Jira Reference (also in PR title):
+- [README](/docs/README.md) updated after adding/changing behaviour of an action
+- Proposed version increment for [release](/docs/README.md#release):
+  - [ ] Patch (bugfix)
+  - [ ] Minor (new feature)
+  - [ ] Major (drop/rename actions)
```

Comment:
```suggestion
  - [ ] Major (breaking changes)
```
More generically, it's breaking changes that are causing major bumps to happen. Whether they're related to an action being dropped, renamed, made incompatible with a previous version, or to reusable workflows instead of actions etc. doesn't really matter.